README.md

```diff
@@ -5,9 +5,21 @@
 High-level declarative visualization library for Python.
 
 This package provides a Python API for building statistical visualizations in a
-declarative manner.
-This API contains no actual visualization rendering code, but instead emits JSON data structures following the [Vega-Lite](https://github.com/vega/vega-lite) specification.
-For convenience, Altair can optionally use [ipyvega](https://github.com/vega/ipyvega) to seamlessly display client-side renderings in the Jupyter notebook.
+declarative manner. By statistical visualization we mean:
+
+* The data source is a data frame that consists of column of different data types
+  that include quantitative, ordinal, nominal and date/time.
+* The data frame is in a [tidy format]() where the rows correspond to samples and
+  the colunms correspond the observed variables.
+* The visual encoding (position, color, size, shape, etc.) of the different
+  columns is intimately related to marginal and conditional probability
+  distributions and relational algebra (in particular `GROUPBY`).
```

Comment:
I think the vocabulary here might scare people – is there a way to re-phrase it?